server/db/db.js

```diff
@@ -46,50 +42,44 @@ var user = {
     _verbose('db.user.create', fxaId, email, oauthToken);
     var query = 'INSERT INTO users (fxaId, email, oauthToken, createdAt) ' +
                 'VALUES ($1, $2, $3, $4)';
-    pg.connect(dbParams, function(err, client, done) {
-      if (err) { return onConnectionError(err, cb); }
-      client.query(query, [fxaId, email, oauthToken, new Date().toJSON()], function(err) {
-        if (err) {
-          log.warn('error saving user: ' + err);
-        } else {
-          _verbose('db.user.create succeeded');
-        }
-        done();
-        cb(err);
-      });
+    var args = [fxaId, email, oauthToken, new Date().toJSON()];
```

Comment:
I left this as `$1`, `$2`...

But we could easily refactor this into something like this:

```js
var query = 'INSERT INTO users (fxaId, email, oauthToken, createdAt) ' +
            'VALUES (:fxaId, :email, :oauthToken, :createdAt)';
var args = {
  fxaId: fxaId,
  email: email,
  oauthToken: oauthToken,
  createdAt: new Date().toJSON()
};
```